README.md

```diff
@@ -11,19 +13,41 @@ For the Points developer take-home assignment, the following task is required: y
     * Calculates and displays the total taxes owed for the salary
     * Displays the amount of taxes owed per band
     * Displays the effective rate
+* There are different levels to this assignment, level 1 being the easiest, level 2 having some gotchas, etc. If you complete a level with ease, feel free to jump to the next one. Please timebox your solution to one day total.
+
+### Level 1
+
+* Fetch the tax rates from this server at: [localhost:5000/tax-calculator/brackets](http://localhost:5000/tax-calculator/brackets)
+
+### Level 2
+
+* Fetch the tax rates by year: [localhost:5000/tax-calculator/brackets/<2019,2020...>](http://localhost:5000/tax-calculator/brackets/2020)
 
-There are different levels to this assignment, level 1 being the easiest, level 2 having some gotchas, etc. If you complete a level with ease, feel free to jump to the next one. Please timebox your solution to one day total.
+_Warning: this endpoint has gotchas_
+
+## Assignment 2: Autonomous Car
+
+### Instructions
+
+* Download and run this server (see [Getting Started](#getting-started))
+* Build an application / tool in React or Python with the following criteria:
+    * Fetches the a autonomous car route from this API server
+    * Takes a set of instructions as input
+    * Determine's if the route was a success or a failure
+    * If it was a failure, determine at which step or position that failed
+* There are different levels to this assignment, level 1 being the easiest, level 2 having some gotchas, etc. If you complete a level with ease, feel free to jump to the next one. Please timebox your solution to one day total.
 
 ### Level 1
 
-* Fetch the tax rates from this server at: http://localhost:5000/tax-calculator/brackets
+* Fetch a random autonomous car route from this server at: [localhost:5000/autonomous-car/routes/](http://localhost:5000/autonomous-car/routes/)
```

Comment:
I finally got around to giving this a proper look, and I'd suggest that for level 1, the route should be static. Otherwise, this assignment may be very confusing and hard to figure out how to accomplish it.
I'd suggest for level 2 to have random routes, as you've built out here.

Reply:
That sounds like a great idea! Just having some difficulty coming up with the endpoint to do this. Now that the endpoint doesn't take in any parameters. 

I'm currently on the train of thought that both the `/tax-calculator` and `/autonomous-car` endpoints should be updated to include a `/level-1/` or `/level-2/` to help separate the different levels.

For example:
 - `/tax-calculator/level-1/brackets/` vs `/tax-calculator/level-2/brackets/year`
 - `/autonomous-car/level-1/routes/` vs `/autonomous-car/level-2/routes/`


What do you think?